package.json

```diff
@@ -13,7 +13,11 @@
     "clean:all": "gulp clean:all",
     "reinstall": "rimraf node_modules && npm --force cache clean && npm install",
     "test:unit": "gulp test:unit",
-    "semantic-release": "semantic-release pre && gulp build && cp -r .git dist && npm publish dist/ && semantic-release post"
+    "semantic-release": "semantic-release pre && gulp build && cp -r .git dist && npm publish dist/ && semantic-release post",
+    "build-planner": "npm install && npm run build && npm pack dist/",
+    "unit-test": "npm run test:unit",
+    "setupFuncTestEnv": "/usr/bin/Xvfb :99 -screen 0 1440x900x24 &",
+    "func-test": "npm run -s setupFuncTestEnv && cd runtime && export API_URL=https://api.prod-preview.openshift.io/api/ && export NODE_ENV=inmemory && npm install && ./tests/run_functional_tests.sh smokeTest"
```

Comment:
Why do you want to export `API_URL` and use `inmemory` mode at the same time?
When we set `NODE_ENV=inmemory` mode, planner uses the `mock` services. It will ignore the `API_URL`.
Also, `run_functional_tests` script always sets `NODE_ENV=inmemory` mode and then starts the planner. You could write this line as
 `npm run -s setupFuncTestEnv && cd runtime && npm install && ./tests/run_functional_tests.sh smokeTest`

Reply:
@jarifibrahim thanks! now, this is what exactly I am trying to find out.
I am just extracting contents from release.groovy to npm targets. It was mentioned here in https://github.com/fabric8-ui/fabric8-planner/blob/master/deploy/release.groovy#L20 
can you please confirm we do not need it this way?
And I am trying your suggestion, let's see.